vmsgen.py

```diff
@@ -619,12 +624,76 @@ def create_unique_op_ids(path_dict):
                 op_id_list.append(op_id_val)
 
 
+def merge_dictionaries(x, y):
+    z = x.copy()   # start with x's keys and values
+    z.update(y)    # modifies z with y's keys and values & returns None
+    return z
+
+
+def remove_query_params(path_dict):
+    """
+    Swagger/Open API specification prohibits describing query parameter as part of the the request mapping url.
+    This method attempts to remove the Query Parameters from the Path and have them in the parameter section.
+     in the Swagger.
+    APIs which result in duplicate APIS on removing the query parameters from the path will be left as is.
```

Comment:
Its not clear what happens when two operations have same path and same http verb. Can you improve the description?

Reply:
The method first checks If removing a query parameter would result in 'absolute duplicate'. i.e same path and HTTP Verb. In that case the query param will NOT be removed from the path.